docs/guide/PINs/README.md

```diff
@@ -53,3 +53,15 @@ A proposal for refactoring environments into `Storage` classes and execution `En
 A proposal for event-driven or long-running flows that run in response to events that arrive in an irregular stream.
 
 **Status:** Proposed
+
+## [PIN-9: Prefect CLI](PIN-9-CLI.md)
+
+A proposal for a flexible and live-updating Prefect Cloud command line client.
+
+**Status:** Proposed
```

Comment:
Could you update this to Accepted since we already merged the first iteration of the CLI into the library?